newIDE/app/src/Profile/Subscription/SubscriptionDetails.js

```diff
@@ -74,7 +74,12 @@ const SubscriptionDetails = ({
       const possiblePlans: Array<PlanDetails> = getSubscriptionPlans().concat(
         getFormerSubscriptionPlans()
       );
-      return possiblePlans.find(plan => plan.planId === subscription.planId);
+      return possiblePlans.find(
+        plan =>
+          subscription.planId &&
+          plan.planId &&
+          subscription.planId.startsWith(plan.planId)
```

Comment:
I'm wondering if this should be a function inside the `Usage` service, there is important logic that may not be 100% clear here (why startsWith?)